### PR TITLE
Allow passing '-' to command line for stdin

### LINKED ISSUE #115: Add ability to pass "-" / stdin
Right now security-checker can only scan the composer.lock file in the current directory.

This is quite inflexible, i.e.:
1. One can't just pass a path to check.
2. It's not possible to pipe in info, e.g. curl https://example.com/composer.lock | security-checker [...] --infile -

Can you add an option to allow passing an input filename?

### PR DESCRIPTION
**Pull request by hannob**
Proposal to fix #115

### CODE REVIEW

File: SensioLabs/Security/SecurityChecker.php [hunk 1]
```diff
@@ -45,10 +45,14 @@ public function check($lock)
             $lock = str_replace('composer.json', 'composer.lock', $lock);
         }
 
-        if (!is_file($lock)) {
+        if (!is_file($lock) && '-' != $lock) {
```
Comment (pected70) [inline SensioLabs/Security/SecurityChecker.php]: !==

File: SensioLabs/Security/SecurityChecker.php [hunk 2]
```diff
@@ -45,10 +45,14 @@ public function check($lock)
...
             throw new RuntimeException('Lock file does not exist.');
         }
 
+        if ('-' == $lock) {
```
Comment (pected70) [inline SensioLabs/Security/SecurityChecker.php]: ===

**Comment by hannob** [comment]
Does this really need documentation?
I feel this is more or less just "behave like most other command line tools".

**Comment by fabpot** [comment]
Closing as this project is not maintained anymore. See https://github.com/fabpot/local-php-security-checker for an open-source software that does the same locally (without any API calls).

**Pull request closed without merging**